Riot/Modules/Room/RoomViewController.m

```diff
@@ -4360,15 +4431,12 @@ - (void)refreshActivitiesViewDisplay
                 // Retrieve the unread messages count
                 NSUInteger unreadCount = self.roomDataSource.room.summary.localUnreadEventCount;
                 
-                self.activitiesViewExpanded = YES;
-                [roomActivitiesView displayScrollToBottomIcon:unreadCount onIconTapGesture:^{
-                    
-                    [self goBackToLive];
-                    
-                }];
+                self.scrollToBottomBadgeLabel.text = unreadCount ? [NSString stringWithFormat:@"%lu", unreadCount] : nil;
```

Comment:
Did you think about using red color when there is a bing message?

Reply:
It's a good remark. I'll propose this change to @gaelledel and create a new PR.